src/main/antora/modules/security/pages/security.adoc

```diff
@@ -1767,28 +1767,31 @@ For this example we'll use a Faces view with a backing bean:
 
 [source,xml]
 ----
+<!DOCTYPE html>
 <html lang="en" xmlns:h="jakarta.faces.html" >
     <h:head/>
     
     <h:body>
-        <h:messages />
+        <h1>
+            <h:messages />
+        </h2>
```

Comment:
This heading doesn't belong here. This should be around 'Login to continue'. And the end tag must match start tag.

Reply:
I just changed the order and made login to continue a header. Can you check again?